internal/hmyapi/blockchain.go

```diff
@@ -63,6 +63,36 @@ func (s *PublicBlockChainAPI) GetBlockByHash(ctx context.Context, blockHash comm
 	return nil, err
 }
 
+// GetCommittee returns committee for a particular epoch.
+func (s *PublicBlockChainAPI) GetCommittee(ctx context.Context, epoch int64) (map[string]interface{}, error) {
+	committee, err := s.b.GetCommittee(big.NewInt(epoch))
+	if err != nil {
+		return nil, err
+	}
+	validators := make([]map[string]interface{}, 0)
+	for _, validator := range committee.NodeList {
+		validatorBalance := new(hexutil.Big)
+		validatorBalance, err = s.b.GetBalance(validator.EcdsaAddress)
+		if err != nil {
```

Comment:
Committee should be a whole. We can't miss a single validator if there is error and return the rest. If any error for a single validator, we should just return error for the api.